content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -14,13 +14,14 @@ Defines a [data source](/concepts/components/prisma-schema/data-sources) <span c
+| ------------------- | -------- | ------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `provider`          | **Yes**  | String (`postgresql`, `mysql`, `sqlite`, `sqlserver`, `mongodb`, `cockroachdb`) | Describes which data source connectors to use.                                                                                                                                                                                                                                                                                                                                                                                                     |
+| `url`               | **Yes**  | String (URL)                                                                    | Connection URL including authentication info. Most connectors use [the syntax provided by the database](/reference/database-reference/connection-urls#format).                                                                                                                                                                                                                                                                                     |
+| `shadowDatabaseUrl` | No       | String (URL)                                                                    | Connection URL to the shadow database used by Prisma Migrate. Allows you to use a cloud-hosted database as the shadow database.                                                                                                                                                                                                                                                                                                                    |
+| `directUrl`         | No       | String (URL)                                                                    | Connection URL for [direct connection to the database](/data-platform/data-proxy/prisma-cli-with-data-proxy#set-a-direct-database-connection-url-in-your-prisma-schema).<br /><br />If you use a connection pooler URL in the `url` argument (for example, if you use the Data Proxy or pgBouncer), Prisma CLI commands that require a direct connection to the database (such as `migrate` or `db pull`) use the URL in the `directUrl` argument. |
```

Comment:
```suggestion
| `directUrl`         | No       | String (URL)                                                                    | Connection URL for [direct connection to the database](/data-platform/data-proxy/prisma-cli-with-data-proxy#set-a-direct-database-connection-url-in-your-prisma-schema).<br /><br />If you use a connection pooler URL in the `url` argument (for example, if you use the Data Proxy or pgBouncer), Prisma CLI commands that require a direct connection to the database (such as `migrate` or `db`) use the URL in the `directUrl` argument. |
```

Reply:
Since we don't specify for migrate

Reply:
I instead decided to go in the opposite direction and list `migrate dev` as an example, but also link out to the full command list